src/Raven.Client/Documents/Operations/Indexes/GetIndexesStatisticsOperation.cs

```diff
@@ -9,17 +9,39 @@ namespace Raven.Client.Documents.Operations.Indexes
 {
     public class GetIndexesStatisticsOperation : IMaintenanceOperation<IndexStats[]>
     {
+        private readonly int? _shard;
+
+        public GetIndexesStatisticsOperation()
+        {
+            
+        }
+
+        public GetIndexesStatisticsOperation(int? shard)
+        {
+            _shard = shard;
+        }
+
         public RavenCommand<IndexStats[]> GetCommand(DocumentConventions conventions, JsonOperationContext context)
         {
-            return new GetIndexesStatisticsCommand();
+            return new GetIndexesStatisticsCommand(_shard);
         }
 
-        private class GetIndexesStatisticsCommand : RavenCommand<IndexStats[]>
+        internal class GetIndexesStatisticsCommand : RavenCommand<IndexStats[]>
```

Comment:
why?

Reply:
reverted

Reply:
reverted (the original idea was to use the command directly without using an operation)